python/sglang/srt/server_args.py

```diff
@@ -108,6 +108,8 @@ class ServerArgs:
     # LoRA
     lora_paths: Optional[List[str]] = None
     max_loras_per_batch: int = 8
+    # Multi-Modal
+    image_processor_max_workers: int = 4
```

Comment:
4 is too small for more powerful machines. Do you have a more adaptive way of setting this value?